release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -170,6 +170,116 @@ See xref:../web_console/creating-quick-start-tutorials.adoc#creating-quick-start
 
 The xref:../support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc#displaying-the-insights-status-in-the-web-console_using-insights-to-identify-issues-with-your-cluster[Insights plug-in] is now integrated into the {product-title} web console. Insights provides cluster health data, such as the number of total issues and total risks of the issues. Risks are labeled as *Critical*, *Important*, *Moderate*, or *Low*. You can quickly navigate to {cloud-redhat-com} for further details about the issues and how to fix them.
 
+[id="ocp-4-7-ibm-z"]
+==== IBM Z and LinuxONE
+
+With this release, IBM Z and LinuxONE are now compatible with {product-title} {product-version}. See xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE] or xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE in a restricted network] for installation instructions.
+
+[discrete]
+===== Notable Enhancements
+
+The following new features are supported on IBM Z and LinuxONE with {product-title} {product-version}:
+
+* KVM on {op-system-base} 8.3 or later is supported as a hypervisor for user-provisioned installation of {product-title} {product-version} on IBM Z and LinuxONE.  See xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[Installing a cluster with {op-system-base} KVM on IBM Z and LinuxONE] for installation instructions.
+* Multipathing
+* OpenShift Pipelines TP
+* Red Hat OpenShift Service Mesh
+* OVN-Kubernetes with an initial installation of {product-title} 4.7
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* z/VM Emulated FBA devices on SCSI disks
+
+[discrete]
+===== Supported features
+
+The following features are also supported on IBM Z and LinuxONE:
+
+* CodeReady Workspaces
+* OpenShift Do (odo)
```

Comment:
Please note, odo name is itself odo and Not OpenShift Do. If you want to embellish it, you can say Developer CLI - odo, but the official name of the product is 'odo'
Also, do we want to mention CodeReady Workspaces in OCP docs. AFAIK, technically they are separately published on the customer portal.

Reply:
If both are correct I would vote for "Developer CLI - odo" 

Regarding CRW, CRC, etc. I think its important for the IBM customers to have an overview what's supported on P/Z in regards to OpenShift. Especially since this constantly changes. 
I had this discussion with @codyhoag in regards to OCS and he said yes add it to the RNs. (Taken it out again because OCS generally isn't supported on 4.7)

Reply:
I think it would be helpful to include things like CRW/CRC here, especially since we're providing such a comprehensive list of supported features. This provides one place where a customer can find supported features for IBM Z and LinuxONE, instead of them being spread around piecemeal, requiring additional searching.

We typically try to keep most features' direct release notes out of the core OCP release notes if they're documented elsewhere, but mentioning cross-product support for an item included in the core OCP release should be fine.

Reply:
Ack @codyhoag makes sense.